src/Discord.Net/Entities/Users/IUser.cs

```diff
@@ -0,0 +1,18 @@
+// todo: docs
+// todo: methods
+using System.Threading.Tasks;
+using Voltaic;
+
+namespace Discord
+{
+    public interface IUser : ISnowflakeEntity
+    {
+        string Username { get; }
+        string Discriminator { get; }
+        // review: does anyone use DiscriminatorValue? seems like a waste of memory
```

Comment:
It is a waste, yes.

Reply:
I can find no usage of it that I wouldn't be happy to replace.

Reply:
Would it be better to make this a ushort and people can just ToString() it if needed? I feel like it could stay consistent since discriminators are numbers.

Reply:
I think that it would be most appropriate to keep Discriminators strings to match the API: https://discordapp.com/developers/docs/resources/user#user-object
There's less risk of things breaking should they change what discriminators can be.